.github/workflows/main.yml

```
@@ -36,33 +110,44 @@ jobs:
         uses: actions/checkout@v2
         with:
           lfs: true
+      - name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: bdist-win-python${{matrix.python-version}}
+          path: dist
       - name: Set up Python ${{matrix.python-version}}
         uses: actions/setup-python@v2
         with:
           python-version: ${{matrix.python-version}}
-      - name: Setup
+      - name: Setup 
```

Super nitpick:
```suggestion
      - name: Setup
```
We should consider adding some yaml file formatting

---

Will do